cmd/copyurl/copyurl.go

```diff
@@ -13,15 +14,17 @@ import (
 )
 
 var (
-	autoFilename = false
-	stdout       = false
-	noClobber    = false
+	autoFilename  = false
+	printFilename = false
+	stdout        = false
+	noClobber     = false
 )
 
 func init() {
 	cmd.Root.AddCommand(commandDefinition)
 	cmdFlags := commandDefinition.Flags()
 	flags.BoolVarP(cmdFlags, &autoFilename, "auto-filename", "a", autoFilename, "Get the file name from the URL and use it for destination file path")
+	flags.BoolVarP(cmdFlags, &printFilename, "print-filename", "p", printFilename, "Print the resulting name from --auto-filename")
```

Comment:
Just a note for the future - we have to be very careful with short flags in sub commands - this means we can't use `-p` as a global flag... In `rsync` it is

        -p, --perms                 preserve permissions

So probably not much use!

We also use `-p` in `rclone tree` also.

I think this use is OK though.

Reply:
Oh, I did not think of that, thanks for the reminder. I can change it if you think that is best, but I were unable to come up with better alternative so no change in latest update... 👎